src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,102 @@
+  | `disable_redis_instrumentation`           | `instrumentation.redis`                   | `instrumentation.redis: disabled`                                                   |
+  | `disable_resque`                          | `instrumentation.resque`                  | `instrumentation.resque: disabled`                                                  |
+  | `disable_sinatra`                         | `instrumentation.sinatra`                 | `instrumentation.sinatra: disabled`                                                 |
+  | `disable_rake`                            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_rake_instrumentation`            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_typhoeus`                        | `instrumentation.typhoeus`                | `instrumentation.typhoeus: disabled`                                                |
+
+
+## Removed Support for Ruby 2.2 and 2.3 [#ruby_2223]
+
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. In order to continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above.
+
+
+## Enabled Thread Tracing by Default [#thread_tracing]
+
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code that occurs inside of threads. In Ruby Agent 9.0, work being done inside of threads will be recorded and associated with the transaction that the thread was created.
+If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
+
+
+## Removed Deprecated Instrumentation [#removed_gem_instrumentation]
+
+  The following gems had been previously deprecated and are now removed.
+
+  * Acts As Solr
+  * Authlogic
+  * DataMapper
+  * Rainbows
+  * Sunspot
+
+
+## Updated API method `set_transaction_name` [#set_transaction_name]
+
+  When the method `NewRelic::Agent.set_transaction_name` is called, it will now always change the name and category of the currently running transaction to what is passed in to the method. This is a change from how it functioned in previous agent versions. Previously, if `set_transaction_name` was called with a new transaction name and a new category that did not match the category that was already assigned to a transaction, neither the new name nor category would be saved to the transaction. 
+  If this method is being called in a situation in which it was previously ignored due to cateogry differences, this will now change the name and category of that transaction. 
```

Comment:
```suggestion
  If this method is being called in a situation in which it was previously ignored due to category differences, this will now change the name and category of the transaction. 
```